rest/model/notifications.go

```diff
@@ -13,17 +12,8 @@ type APIEventStats struct {
 	PendingNotificationsByType apiNotificationStats `json:"pending_notifications_by_type"`
 }
 
-func (n *APIEventStats) BuildFromService(h interface{}) error {
-	stats, ok := h.(*notification.NotificationStats)
-	if !ok {
-		return errors.Errorf("programmatic error: expected notification stats but got type %T", h)
-	}
-
-	return n.PendingNotificationsByType.BuildFromService(stats)
-}
-
-func (n *APIEventStats) ToService() (interface{}, error) {
-	return nil, errors.New("(*APIEventStats) ToService not implemented")
+func (n *APIEventStats) BuildFromService(stats notification.NotificationStats) {
+	n.PendingNotificationsByType.BuildFromService(stats)
```

Comment:
Should we implement `ToService` ?

Reply:
I think there's no point in implementing functions that aren't used anywhere, since we aren't trying to implement the interface.

Reply:
Fair, sgtm